include/Basis/SingleParticleBasis.hpp

```diff
@@ -330,7 +330,7 @@ class SingleParticleBasis {
      *
```

Comment:
Could you change the previous line to: "indices of the AOs used in the Mulliken partitioning scheme"?